### ISSUE TITLE
Error when installing

### ISSUE THREAD

**Issue by PythonNube** [Sennevds/system_sensors#65]
I get the following error when I try to install.

  Could not find a version that satisfies the requirement rpi_bad_power==0.0.3 (from -r requirements.txt (line 5)) (from versions: 0.1.0)
No matching distribution found for rpi_bad_power==0.0.3 (from -r requirements.txt (line 5))

I tried to install that version and I get this error.
pi@raspberrypi:~/system_sensors $ pip3 install rpi_bad_power==0.0.3
Collecting rpi_bad_power==0.0.3
  Could not find a version that satisfies the requirement rpi_bad_power==0.0.3 (from versions: 0.1.0)
No matching distribution found for rpi_bad_power==0.0.3

I also tried to update the requirments file to the latest version 0.1.0 and the install works but when I run it I get this error

pi@raspberrypi:~/system_sensors $ python3 src/system_sensors.py src/settings.yaml
  File "src/system_sensors.py", line 103
    + f'"temperature": {get_temp()},'
                                    ^
SyntaxError: invalid syntax

I am running raspbian on a PI 3B+

PRETTY_NAME="Raspbian GNU/Linux 9 (stretch)"
NAME="Raspbian GNU/Linux"
VERSION_ID="9"
VERSION="9 (stretch)"
VERSION_CODENAME=stretch

Python version
pi@raspberrypi:~/system_sensors $ python3 --version
Python 3.5.3

**Comment by Sennevds** [comment]
As stated here https://github.com/Sennevds/system_sensors you need at least python 3.6

**Comment by PythonNube** [comment]
I upgraded to version 3.7
pi@raspberrypi:~/system_sensors $ python3 --version
Python 3.7.0

I tried re-installing but still get the error about the rpi_bad_power.  I updated the requirements file to version 0.1.0 and did the install but now I get the below error.

l
Traceback (most recent call last):
  File "src/system_sensors.py", line 11, in <module>
    from rpi_bad_power import new_under_voltage
ModuleNotFoundError: No module named 'rpi_bad_power'


Sorry,  I am obviously not an expert.  Just hope to get this working.  I installed this on another PI and had no issues so I am not sure what is going on here.
